tuf/client/updater.py

```diff
@@ -1379,12 +1379,50 @@ def verify_target_file(target_file_object):
 
 
 
-  def _verify_uncompressed_metadata_file(self, metadata_file_object,
+  def _verify_root_sigs(self, signable):
```

Comment:
I think we need a better name, like `_self_verify_root_sigs()`, because it wasn't clear where the keys are coming from (the new root metadata file itself, which seems dangerous until you consider that this should be called only _after_ verifying that new follows from old, then you check that new follows from new).

Reply:
`_verify_root_self_signed()`?

Reply:
good suggestion, I like it